xml/ns-System.Numerics.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Numerics">
   <Docs>
-    <summary>The <see cref="N:System.Numerics" /> namespace contains numeric types that complement the numeric primitives, such as <see cref="T:System.Byte" />, <see cref="T:System.Double" />, and <see cref="T:System.Int32" />, that are defined by the .NET Framework.</summary>
+    <summary>Provides a set of SIMD-enabled vector types.</summary>
```

Comment:
@mairaw, Yes, the original one. Except for one type not in the .NET Framework, the .NET Framework SIMD documentation is identical to the .NET Core documentation.